hash/zx81_cass.xml

```diff
@@ -5346,6 +5461,24 @@ RAND USR 16530
 		</part>
 	</software>
 
+	<software name="panique">
+		<description>Panique</description>
+		<year>1983</year>
+		<publisher>ERE Informatique</publisher>
+		<info name="language" value="French" />
+		<info name="usage" value="16K" />
+		<part name="cass1" interface="zx81_cass">
+			<dataarea name="cass" size="10442">
+				<rom name="Panique - Side 1.tzx" size="10442" crc="d35b9d2d" sha1="7c0c96249c38ebd2174849846f0dbe8c637a065d"/>
+			</dataarea>
+		</part>
+		<part name="cass2" interface="zx81_cass">
+			<dataarea name="cass" size="9951">
+				<rom name="Panique - Side 2.tzx" size="9951" crc="22e7c98f" sha1="5303ef13479064390f45d63478161f48c6dc94ad"/>
+			</dataarea>
+		</part>
```

Comment:
Can we trust the “Side 1” and “Side 2” enough to use them for `part_id`?  I haven’t been able to find a picture of the cassette, only the inlay.

Reply:
https://www.ebay.fr/itm/186682844499 this listing has a photo, it shows "B".

Reply:
I can add the part_id and rename the roms to Side A and Side B on next PR.
![image](https://github.com/user-attachments/assets/ab8c60d0-4310-493a-96dd-5c2fca57e3bd)
![image](https://github.com/user-attachments/assets/5fdecf52-4c62-4cca-b0f5-51278ba1dc43)